<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CHeaterFaultInfo</class>
 <widget class="QWidget" name="CHeaterFaultInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>531</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>加热器群组：</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QComboBox" name="heaterGroupSwith">
      <property name="font">
       <font>
        <family>宋体</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <item>
       <property name="text">
        <string>机舱</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>塔底</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>531</width>
     <height>51</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>529</width>
      <height>49</height>
     </rect>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>1#加热器</string>
     </property>
    </widget>
    <widget class="QLabel" name="faultInfo_0">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>441</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>设备正常</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QScrollArea" name="scrollArea_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>531</width>
     <height>51</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>529</width>
      <height>49</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>2#加热器</string>
     </property>
    </widget>
    <widget class="QLabel" name="faultInfo_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>441</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>设备正常</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QScrollArea" name="scrollArea_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>531</width>
     <height>51</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>529</width>
      <height>49</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>3#加热器</string>
     </property>
    </widget>
    <widget class="QLabel" name="faultInfo_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>511</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>设备正常</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QScrollArea" name="scrollArea_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>531</width>
     <height>51</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>529</width>
      <height>49</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>4#加热器</string>
     </property>
    </widget>
    <widget class="QLabel" name="faultInfo_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>441</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>设备正常</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QScrollArea" name="scrollArea_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>531</width>
     <height>51</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>529</width>
      <height>49</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>54</width>
       <height>12</height>
      </rect>
     </property>
     <property name="text">
      <string>5#加热器</string>
     </property>
    </widget>
    <widget class="QLabel" name="faultInfo_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>441</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>设备正常</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
